firebase-storage/src/main/java/com/google/firebase/storage/internal/ActivityLifecycleListener.java

```diff
@@ -33,6 +34,7 @@
  * @hide
  */
 @SuppressWarnings("JavaDoc")
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Rather than adding this to every class, can we put it on the package?  e.g. add a package-info.java in .../internal/ with:

```
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
package com.google.firebase.storage.internal;
```

I don't want to have to remember to put this on every class if possible.

Reply:
Yes, that works as well. Updated the PR.